### ISSUE TITLE
Accessibility error: ARIA hidden element must not contain focusable elements

### ISSUE THREAD

**Issue by Ke1sy** [react-component/dialog#287]
**Description:**
The issue is that the div elements are focusable and have the attribute "aria-hidden="true" at the same time.
![image](https://user-images.githubusercontent.com/26119263/164415933-8221dfe0-0f54-4030-94fd-3d4e44a36214.png)
![image](https://user-images.githubusercontent.com/26119263/164416526-c47576fc-136e-446c-8807-6dd051b7720c.png)
**Tools:**
axe Devtools, Chrome
**Proposed solution:**
When the dialog is visible: set _"aria-hidden="false"_ and _tabIndex="0"_
When the dialog is not visible: set _"aria-hidden="true"_ and _tabIndex="-1"_
